go.mod

```diff
@@ -14,7 +14,9 @@ require (
 	github.com/ipfs/go-log/v2 v2.5.1
 	github.com/klauspost/compress v1.17.11
 	github.com/libp2p/go-libp2p v0.37.2
+	github.com/libp2p/go-libp2p-kad-dht v0.25.2
```

Comment:
The deps here make me consider breaking out f3 command into its own sub-module.
On the other hand, dep trimming should solve it for most use cases.

Reply:
Yeah in later version of go this is a non-issue?

Dependency tree in this repo could be much thinner in anycase. Observer dependency to duckdb is another example.